.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: check
-          args: --workspace --all-targets
+          args: --workspace --all-targets --all-features
```

Comment:
While the features can technically be used in parallel, should we `check` the possible combinations in isolation? (ie `linked`, `loaded`, and `linked,loaded`)

Reply:
They don't really interact, and in general testing every combination of features is combinatorically infeasible, so my inclination is not to bother.

Reply:
There are only two features that are intended to be used separately, but fine to leave as-is.